packages/askui-nodejs/src/execution/inference-client.ts

```diff
@@ -50,9 +55,9 @@ export class InferenceClient {
       instruction,
       customElements,
     };
-    const url = urljoin(this.url, 'predict-command');
-    const httpResponse = await this.httpClient.post<ControlCommand>(url, httpBody);
-    return ControlCommand.fromJson(httpResponse, resizedImage.resizeRatio);
+    const url = urljoin(this.url, 'inference');
+    const httpResponse = await this.httpClient.post<CommandData>(url, httpBody);
+    return CommandData.getControlCommandFromJson(httpResponse, resizedImage.resizeRatio);
```

Comment:
can you explain to me why you added this? 
was it not possible to use `ControlCommand.fromJson(httpResponse.data, resizedImage.resizeRatio);`

Reply:
Was also thinking to just add data to ContolCommand but because ControlComand is part of this data array I think its easier to model this and also the new model has the type we added to the respone